platform/nebulas/client.go

```diff
@@ -0,0 +1,44 @@
+package nebulas
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type Client struct {
+	HTTPClient *http.Client
+	URL        string
+}
+
+func (c *Client) GetTxs(address string, page int) ([]Transaction, error) {
+
+	uri := fmt.Sprintf("%s/tx?a=%s&p=%d",
+		c.URL,
```

Comment:
Can you check Binance or waves client? We have a special method to request data easily